### PR TITLE
Fixed navbar linking issue #99 and added performance improvements

### PR DESCRIPTION
**Pull request by Sid911**
# Changes done in this Pull Request

- Github pages link : https://sid911.github.io/gci18/
- Fixes: #99
- Added `preload="metadata"` to the videos, so that it doesn't hamper the loading times 
- Added link to the Projects page

## Description / Changes

The navbar is now functional and the loading time has been improved.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
               <li><a href="#welcome">Welcome</a></li>
               <li><a href="#mentors">Mentors</a></li>
               <li><a href="#students">Students</a></li>
-              <li><a href="#projects">Projects</a></li>
+              <li><a href="/projects.html">Projects</a></li>
```
Comment (fragm3) [inline _includes/header.html]: Why /project.html? Correct it.

Comment (jogendra) [inline _includes/header.html]: There will not be any different page for Projects, there will be a section for project within this page.

**Comment by Sid911** [comment]
IS it not the student's projects? @fragm3

**Comment by fragm3** [comment]
@Sid911 its supposed to be a section like Mentors, do those changes and we are good to merge this.

**Comment by Sid911** [comment]
working on it 😊

**Comment by Sid911** [comment]
@fragm3  done

**Comment by Sid911** [comment]
I added `/index/#section_name` so that the place is visitable from different files like that students project page

**Comment by fragm3** [comment]
@Sid911 why have you add '/' in front, it wont work if you put '/'

File: _includes/header.html [hunk 2]
```diff
@@ -36,10 +36,10 @@
         <div class="module-group right">
           <div class="module left">
             <ul class="menu">
-              <li><a href="#welcome">Welcome</a></li>
-              <li><a href="#mentors">Mentors</a></li>
-              <li><a href="#students">Students</a></li>
-              <li><a href="#projects">Projects</a></li>
+              <li><a href="/#welcome">Welcome</a></li>
```
Comment (fragm3) [inline _includes/header.html]: only "welcome"(no /), same for below 3 lines

**Comment by Sid911** [comment]
done😊.........

**Comment by akshatnitd** [comment]
@fragm3 This PR was merged without a minimum of 2 approvals. Also, please ensure to approve yourself before merging. Thanks

**Pull request merged**